### PR TITLE
WEBSITE - Encourage users on the Community page to ask questions on S…

### PR DESCRIPTION
**Pull request by ErickRamirezAU**
…tack Overflow/Exchange

patch by Erick Ramirez; reviewed by ### for CASSANDRA-17852

Co-authored by: Erick Ramirez <erickramirezau@apache.org>

### CODE REVIEW

File: site-content/source/modules/ROOT/pages/community.adoc [hunk 1]
```diff
@@ -45,70 +45,94 @@ Read the https://www.apache.org/foundation/policies/conduct[Apache Code of Condu
 == Discussions
 If you are a Cassandra user or want to learn more, we invite you to join these channels created specifically for end user questions and discussions.
```
Comment (ekaterinadimitrova2) [inline site-content/source/modules/ROOT/pages/community.adoc]: "If you are a Cassandra user or if you want to learn more"

File: site-content/source/modules/ROOT/pages/community.adoc [hunk 2]
```diff
@@ -45,70 +45,94 @@ Read the https://www.apache.org/foundation/policies/conduct[Apache Code of Condu
...
 ----------
 [discrete]
-=== User Mailing List
+=== Q&A Forums
 ----------
 
 [openblock, float66 full-800]
 ----------
-General mailing list for user questions and discussions. This is also where new releases are announced.
+For questions about developing apps or need help operating Cassandra, post on the Stack Exchange network.
```
Comment (ekaterinadimitrova2) [inline site-content/source/modules/ROOT/pages/community.adoc]: Stack network or we need to move both sites

**Review by ekaterinadimitrova2 — COMMENTED** [review]
Overall LGTM

File: site-content/source/modules/ROOT/pages/community.adoc (hunk 1 above)
Comment (ErickRamirezAU) [reply site-content/source/modules/ROOT/pages/community.adoc]: Updated 👍

File: site-content/source/modules/ROOT/pages/community.adoc (hunk 2 above)
Comment (ErickRamirezAU) [reply site-content/source/modules/ROOT/pages/community.adoc]: Updated to `Stack network`. 👍

**Pull request merged**